cypress/integration/404.spec.ts

```diff
@@ -0,0 +1,23 @@
+it('should display custom 404 page when not found', () => {
+  cy.visit('/not-existing-page', { failOnStatusCode: false })
+
+  cy.title().should('equal', '404 - Not Found | Octoclairvoyant')
+
+  cy.findByRole('heading', {
+    name: 'This page could not be found.',
+  }).should('exist')
+
+  cy.findByText('Octoclairvoyant cannot divine the page you wanted.').should(
+    'exist'
+  )
+
+  cy.findByRole('button', { name: 'Go to comparator' }).click()
+  cy.url().should('equal', `${Cypress.config().baseUrl}/comparator`)
```

Comment:
I'm seeing: `${Cypress.config().baseURL}/comparator`in the test. 
In my tests I used another way to define the URL.

Is your way the best way to define the URL?

Reply:
This way is better, so it always takes the `baseUrl` defined in `cypress.json`. Otherwise, if we hardcode the base URL and it's changed someday, all tests would start failing. I changed your test to use this one btw.